### DISCUSSION TITLE
Raspberry Pi 3B+ - Power doesn't stay off

### DISCUSSION THREAD

**Discussion by pd1701** [mvp/uhubctl#533] [General]
I'm using uhubctrl on OpenWRT on a Pi 3B+.  Using the correct command as listed in the FAQ, the USB power does turn off, but then turns back on.  That is using the parameter -a 0.  Is this a known bug, or a limitation of the Pi 3B+?

**Comment by mvp** [comment]
1. Make sure you use uhubctl version 2.4.0 or later
2. Read FAQ https://github.com/mvp/uhubctl#power-comes-back-on-after-few-seconds-on-linux

**Reply by pd1701** [reply]
Thank you, the -r paramater fixed it for me.
